css/css-text/overflow-wrap/overflow-wrap-min-content-size-001.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Test: overflow-wrap: break-word and intrinsic sizing</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#overflow-wrap-property">
+<meta name="flags" content="">
+<link rel="match" href="reference/overflow-wrap-min-content-size-001-ref.html">
+<meta name="assert" content="Soft wrap opportunities introduced by overflow-wrap:break-word **are** considered when calculating min-content intrinsic sizes.">
+<style>
+table {
```

Comment:
I think the 'table' object here is a distraction. Perhaps i'm not understanding it properly, but at least in chrome, the max-width properly is not applied and 'table' layout just honors the min-content to determine its width. Wouldn't be better to use an inline-block, a float or just a div with 'min-content' width ?

Reply:
This is a response to this comment as well as the next one:

Yes, with `width: min-content` on the div directly, without using a table, would achieve the goal of this test, and be simpler. However, `width: min-content` is not yet widely supported, and tests should not depend on features that are not yet stable.

Using a table, and forcing it to be as small as possible will achieve min-content sizing of the div in all browsers.

I'll add a comment to the test saying that.